dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/UploadFrameworkDataJourney.ts

```diff
@@ -21,6 +21,7 @@ describe("As a user, I expect the dataset upload process to behave as I expect",
     "",
     {
       executionEnvironments: ["developmentLocal", "ci", "developmentCd"],
+      onlyExecuteOnReset: false,
```

Comment:
"OnReset" is a bit ambiguous I'd prefer "OnDatabaseWipe" or similar

Reply:
Agree on the name, changed it to `...OnDatabaseReset`, wipe sounds informal IMO. For the rest I refer to my other answer.